pkg/model/project.go

```diff
@@ -234,24 +233,22 @@ func (p *ProjectSSOConfig) Update(sso *ProjectSSOConfig) error {
 
 // Encrypt encrypts sensitive data in ProjectSSOConfig.
 func (p *ProjectSSOConfig) Encrypt(encrypter encrypter) error {
-	if p.Github != nil {
-		if err := p.Github.Encrypt(encrypter); err != nil {
-			return err
-		}
+	if p.Github == nil {
+		return nil
 	}
-	if p.Google != nil {
+	if err := p.Github.Encrypt(encrypter); err != nil {
+		return err
 	}
 	return nil
```

Comment:
```suggestion
	return p.Github.Encrypt(encrypter)
```

Reply:
Thanks for the review!
[Fix return error by function](https://github.com/pipe-cd/pipecd/pull/4622/commits/d3086a077043b56e571c06b565ca99b6b8e63f75)